docs/src/content/docs/getting-started.mdx

```diff
@@ -50,11 +52,24 @@ Try Starlight in your browser:
 
 ## Creating content with Starlight
 
-Starlight supports authoring content in Markdown and MDX.
+Starlight is ready for you to add new content, or bring your existing files!
+
+### File formats
+
+Starlight supports authoring content in Markdown and MDX. (There is experimental support for Markdoc available via the [Astro Markdoc integration](https://docs.astro.build/en/guides/integrations-guide/markdoc/).)
```

Comment:
Suggestion to rephrase slightly:

```suggestion
Starlight supports authoring content in Markdown and MDX. (You can add support for Markdoc by adding the experimental [Astro Markdoc integration](https://docs.astro.build/en/guides/integrations-guide/markdoc/).)
```

Reply:
If you like this edit (or this + tweaks of your own), this PRs all yours to merge!